cmd/relayer/main.go

```diff
@@ -118,10 +119,10 @@ var (
 			EnvVars: []string{"SWAPD_BOOTNODES"},
 		},
 		&cli.StringFlag{
-			Name: flagRelayerCommission,
-			Usage: "Minimum commission percentage (of the swap value) to receive:" +
-				" eg. --relayer-commission=0.01 for 1% commission",
-			Value: common.DefaultRelayerCommission.Text('f'),
+			Name: flagRelayerFee,
+			Usage: "Minimum commission fee to receive in ETH:" +
+				" eg. --relayer-fee=0.01 for a fee of 0.01 ETH",
```

Comment:
I think we need slightly different wording so the user understands that this is a filter, under which the relayer rejects the request. The word "receive" sounds passive. Maybe something like this?
```
Min fee to accept a request: eg. --relayer-fee=0.01 rejects requests under 0.01 ETH
```